dask/dataframe/io/tests/test_csv.py

```diff
@@ -1275,3 +1275,21 @@ def test_to_csv_header(header, header_first_partition_only, expected_first, expe
             line = fp.readline()
             assert line == expected_next
         os.remove(filename)
+
+def test_to_csv_line_ending():
+    df = pd.DataFrame({'x': [0]})
+    ddf = dd.from_pandas(df, npartitions=1)
+    expected = {b'0\r\n', b'0\n'} # either/or
```

Comment:
Alternatively, we could have two tests, one for unix and one for windows, and each asserts the correct line ending and is skipped when it's not that platform. What do you think? Something like

```
pytest.mark.parametrize('line_ending', [
    pytest.param('0\r\n', marks=[pytest.mark.skipif('os.name == "nt"')],
    pytest.param(...)
])
```

Reply:
Well, I was thinking of determining the expected output based on `os.linesep`, which I *think* would have the same effect. But then I was thinking that the built-in csv module *always* terminates records with `\r\n`, so I wasn't sure which to expect on Unix. The test as written certainly captures the behavior I initially reported on Windows, but I can definitely modify the test(s) if I haven't persuaded you that it's not necessary :-) (I haven't fully convinced myself one way or the other)

Reply:
I see, thanks. Your current setup seems OK.